ui/app/AppLayouts/Communities/popups/RemotelyDestructPopup.qml

```diff
@@ -63,12 +70,56 @@ StatusDialog {
     implicitWidth: 600 // by design
     padding: 0
 
-    contentItem: TokenHoldersPanel {
-        id: tokenHoldersPanel
-        tokenName: root.collectibleName
-        isSelectorMode: true
-        onSelfDestructAmountChanged: d.updateTokensToDestruct(walletAddress, amount)
-        onSelfDestructRemoved: d.clearTokensToDesctruct(walletAddress)
+    contentItem: ColumnLayout {
+        spacing: 16
+        TokenHoldersPanel {
+            id: tokenHoldersPanel
+            tokenName: root.collectibleName
+            Layout.fillWidth: true
+            Layout.fillHeight: true
+            isSelectorMode: true
+            onSelfDestructAmountChanged: d.updateTokensToDestruct(walletAddress, amount)
+            onSelfDestructRemoved: d.clearTokensToDesctruct(walletAddress)
+        }
+        FeesBox {
+            Layout.fillWidth: true
+            Layout.bottomMargin: 16
+            Layout.leftMargin: 16
+            Layout.rightMargin: 16
+            implicitWidth: 0
+            accountErrorText: root.feeErrorText
+            placeholderText: root.feePlaceholderText
+            showAccountsSelector: true
+            model: d.tokenCount > 0 ? singleFeeModel : undefined
+            accountsSelector.model: SortFilterProxyModel {
```

Comment:
Also, please rebase on top of master and you will be able to remove this SFPM and just call to root.accounts.